ansible/roles/kubernetes/tasks/secrets.yml

```diff
@@ -81,15 +81,41 @@
 
 - include: gen_tokens.yml
   when: inventory_hostname == groups['masters'][0]
```

Comment:
@xialonglee The `know_tokens.csv` file has to same on all masters and the `*.token` files are copied too, so when changing the "first" master it won't regenerate the tokens and make the running cluster unusable because of the new tokens for the components.